extension/content.css

```diff
@@ -399,21 +399,52 @@ svg.octicon.octicon-star.dashboard-event-icon {
 }
 
 .reaction-summary-item div,
+.reaction-summary-item div a,
 .reaction-summary-item div img {
 	display: inline-block;
 }
 
+.reaction-summary-item div a {
+	transition: margin-left 0.2s;
+	padding-left: 3px;
+}
+
 .reaction-summary-item div img,
 .reaction-summary-item div span {
 	width: 20px;
 	height: 20px;
-	margin-left: 3px;
 	vertical-align: middle;
 }
+
 .reaction-summary-item div img {
+	color: #FFF; /* affects box-shadow */
+	box-shadow: 0 0 0 2px;
 	border-radius: 3px;
 }
 
+.reaction-summary-item.user-has-reacted div img {
+	color: #f2f8fa;
+}
+
+/* overlap reaction avatars when there are 5+ types of reactions */
+.reaction-summary-item:first-child:nth-last-child(n+5) a:nth-child(n+2),
+.reaction-summary-item:first-child:nth-last-child(n+5) ~ * a:nth-child(n+2) {
+	padding-left: 0;
```

Comment:
Please look at these two screenshots. "Normally" the avatars are spaced ~5px apart. During the `:hover` they are spaced ~2px.

**"Normal" spacing:**
![screen shot 2017-03-06 at 10 38 28 am](https://cloud.githubusercontent.com/assets/737065/23616868/24eb54b6-0259-11e7-9000-22fbd1ec2211.png)

**`:hover` spacing:**
![screen shot 2017-03-06 at 10 38 19 am](https://cloud.githubusercontent.com/assets/737065/23616867/24ea7d52-0259-11e7-9df4-4662cf31b2bb.png)